posthog/errors.py

```diff
@@ -95,10 +95,15 @@ class CHQueryErrorTooManySimultaneousQueries(InternalCHQueryError):
     pass
 
 
+class CHQueryErrorCannotScheduleTask(InternalCHQueryError):
```

Comment:
our CH errors are dynamically typed, but we can add them here when we need to reference them elsewhere in the code and there needs to be a concrete type